nodeadm/internal/containerd/base_runtime_spec.go

```diff
@@ -0,0 +1,44 @@
+package containerd
+
+import (
+	_ "embed"
+	"encoding/json"
+	"strings"
+
+	"github.com/awslabs/amazon-eks-ami/nodeadm/internal/api"
+	"github.com/awslabs/amazon-eks-ami/nodeadm/internal/util"
+	"go.uber.org/zap"
+)
+
+const containerdBaseRuntimeSpecFile = "/etc/containerd/base-runtime-spec.json"
+
+//go:embed base-runtime-spec.json
+var baseRuntimeSpecData string
+
+var baseRuntimeSpecMap api.InlineDocument
+
+func init() {
+	if err := json.Unmarshal([]byte(baseRuntimeSpecData), &baseRuntimeSpecMap); err != nil {
+		panic(err)
+	}
+}
+
+func writeBaseRuntimeSpec(cfg *api.NodeConfig) error {
+	zap.L().Info("Writing containerd base runtime spec...", zap.String("path", containerdBaseRuntimeSpecFile))
+	baseRuntimeSpecData := baseRuntimeSpecData
```

Comment:
we end up pulling this from the global scope anyway so should you just move the init code inside here?

Reply:
Sure, done